<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(162, 103, 172);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="main_pg">
       <widget class="QPushButton" name="break_btn">
        <property name="geometry">
         <rect>
          <x>1300</x>
          <y>850</y>
          <width>496</width>
          <height>106</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 56pt &quot;Calibri&quot;;
background-color: rgb(206, 123, 176);
color:rgb(255, 188, 209);
border-radius: 20px;
</string>
        </property>
        <property name="text">
         <string>Break</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_btn">
        <property name="geometry">
         <rect>
          <x>714</x>
          <y>672</y>
          <width>492</width>
          <height>96</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 56pt &quot;Calibri&quot;;
background-color: rgb(206, 123, 176);
color:rgb(255, 188, 209);
border-radius: 20px;
</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="money_edit">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>550</y>
          <width>680</width>
          <height>106</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Enter money</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 70pt &quot;Calibri&quot;;
color:rgb(255, 188, 209);
border-radius: 20px;
background-color: rgb(206, 123, 176);</string>
        </property>
        <property name="text">
         <string>Money</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="piggy_icon">
        <property name="geometry">
         <rect>
          <x>804</x>
          <y>155</y>
          <width>312</width>
          <height>312</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/piggy_icon/piggy.png) no-repeat center center fixed;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="savings_pg">
       <widget class="QLabel" name="savings_label">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>300</y>
          <width>664</width>
          <height>158</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 100pt &quot;Calibri&quot;;
color:rgb(255, 188, 209);</string>
        </property>
        <property name="text">
         <string>Your savings</string>
        </property>
       </widget>
       <widget class="QLabel" name="money_label">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>496</y>
          <width>664</width>
          <height>175</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 100pt &quot;Calibri&quot;;
color:rgb(255, 188, 209);</string>
        </property>
        <property name="text">
         <string>____</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
